docker-compose.override.yml

```diff
@@ -97,19 +104,22 @@ services:
         SPE_RESOURCES_IMAGE: ${SPE_RESOURCES_IMAGE}
 
   solr:
+    restart: ${DOCKER_RESTART}
     volumes:            
       - ${LOCAL_DATA_PATH}\solr:c:\data  
     mem_limit: ${MEM_LIMIT_SOLR:-1GB}  
 
   solr-init:
     image: ${MVP_DOCKER_REGISTRY}mvp-solr-init:${SITECORE_VERSION}
+    restart: ${DOCKER_RESTART}
```

Comment:
Do we want to restart the SOLR Init container job? 

This should terminate once the job is complete.

Reply:
Sure, will check and update....